samples/general_assignments/codeval.ini

```diff
@@ -0,0 +1,6 @@
+[SERVER]
```

Comment:
perhaps we can just have one example ini file and note the fields that are only needed in the distributed case.

Reply:
Done [here](https://github.com/SJSU-CS-systems-group/CodEval/pull/12)